malariagen_data/pf7.py

```diff
@@ -0,0 +1,314 @@
+import json
+import os
+
+import dask.array as da
+import pandas as pd
+import xarray
+import zarr
+
+from malariagen_data.util import (
+    DIM_ALLELE,
+    DIM_PLOIDY,
+    DIM_SAMPLE,
+    DIM_VARIANT,
+    da_from_zarr,
+    init_filesystem,
+    init_zarr_store,
+)
+
+DIM_ALT_ALLELE = "alt_alleles"
+DIM_STATISTICS = "sb_statistics"
+DIM_GENOTYPES = "genotypes"
+
+
+class Pf7:
+    """Provides access to data from the Pf7 release.
+
+    Parameters
+    ----------
+        url (str, optional): Base path to data. Default uses Google Cloud Storage "gs://pf7_release/",
+                             or specify a local path on your file system if data have been downloaded.
+        data_config (str, optional) : path to config for structure of Pf7 data resource. Defaults to config in repository.
```

Comment:
Please edit to match [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#parameters) style.

Reply:
Should now all be in numpydoc style